test/time.test.js

```diff
@@ -0,0 +1,24 @@
+import test from 'blue-tape'
+import * as time from '../src/time'
+import R from 'ramda'
+
+test('convertSecondsToTime()', t => {
+  const testcases = [
```

Comment:
The same comment as in https://github.com/cvut/fittable/commit/64ae07c6a88b001e655000498f8b64d492d18914#commitcomment-14958484 …